CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+cmake_minimum_required(VERSION 3.0.0)
```

Comment:
I am not sure if it's a good idea to add the CMakeLists.txt.... We could use the existing qmake based project. Having 2 indipendent build systems will inevitable cause one or the other to get outdated as it requires any change to be duplicated. While indeed qmake is deprecated and cmake the new way forward with Qt6, right now I don't have the time to switch android and iOS builds to cmake.

Reply:
That means I can build it without CMakeLists.txt as well?

Reply:
yes, sure, With the .pro file

Reply:
You'd need a clickable.json file with "builder": "qmake", similar to the from nymea-app: https://github.com/nymea/nymea-app/blob/master/packaging/ubuntu/click/clickable.json

Reply:
I believe you'd only need the "builder" and perhaps the "qtversion" entries though...  afaict berrylan does not requrie any dependencies_target... Not sure...